### DISCUSSION TITLE
Class Recordings?

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#556] [General]
I'm trying to get myself caught up and class this past week was really helpful @nquarder. When will the recording be uploaded to canvas? Weekends can be really productive, but the recordings help me to remember where I left off!

**Comment by nquarder** [comment]
Hey Randi! I'm cc'ing @eculler here too as she's the one who usually posts those to Canvas. Elsa can you help with this when you get a chance? Thanks team! -NQ

**Comment by nquarder** [comment]
Hey @BioNCphilic - It looks like the recordings from last week are in Canvas (just not under Modules, yet). In Canvas if you click on Zoom and then select Cloud Recordings, you'll see the classes from last Weds Oct 9 are up. 

<img width="2230" alt="Screen Shot 2024-10-14 at 9 49 21 AM" src="https://github.com/user-attachments/assets/12ff8ed0-e659-4831-8c42-3aa0d8c2b1e5">

I'm not actually seeing the recordings from Thurs Oct 10 anywhere but we'll do a bit more digging around (cc: @eculler). And we'll get the links added to Modules sometime today or tomorrow. 

Not sure if this link will take you there directly or if you need to navigate from your end but give it a try: 
https://canvas.colorado.edu/courses/109233/external_tools/16942


Thanks for your patience! -NQ

**Reply by BioNCphilic** [reply]
Thanks. Apparently I don't have permission to view the recording from the zoom directory.

**Comment by eculler** [comment]
@BioNCphilic we're all caught up now -- sorry for the delay!

**Reply by BioNCphilic** [reply]
Thanks. No worries. I used the down time to spruce up my portfolio with a theme.
